README.md

```diff
@@ -65,7 +215,20 @@ gsutil -m cp ./*Nov*.txt gs://httparchive/crawls_manifest/
 - Dataflow temporary and staging artifacts in GCS
 - BigQuery (final landing zone)
 
-## Temp table cleanup
+## Builds and Deployments
+
+The data pipeline uses Cloud Build to create Dataflow flex templates and upload them to Artifact Registry and Google Cloud Storage
+- Cloud Build [linked here](https://console.cloud.google.com/cloud-build/builds?project=httparchive)
+- Artifact Registry images [linked here](https://console.cloud.google.com/artifacts/docker/httparchive/us-west1/data-pipeline?project=httparchive)
+- Flex tempaltes in GCS [gs://httparchive/dataflow/templates](https://console.cloud.google.com/storage/browser/httparchive/dataflow/templates?project=httparchive)
+
+Dataflow flex templates are built using the `build_flex_template.sh` script. GCP Workflows must be udpated to the latest build tag manually to take effect in production.
```

Comment:
Updating the workflow is accomplished just by running the script - there's no additional action needed?

I'm also a bit confused by the use of "manually" as it seems the GH actions will do that automatically for merges into `main`.

Reply:
The script was written first, then the GHA was added as a convenience/automation. I'll add a clarification here to explain that updating the workflow and flex template can be done manually via the scripts, or automatically via GHA.